Bluepill-cli/Bluepill-cli/Bluepill/Bluepill.m

```diff
@@ -249,7 +253,6 @@ - (void)createSimulatorWithContext:(BPExecutionContext *)context {
         [BPUtils printInfo:ERROR withString:[@"Timeout: " stringByAppendingString:stepName]];
     };
 
-    self.maxInstallTries = [self.config.maxInstallTries integerValue];
```

Comment:
This is because GitHub uses `git diff master...branch` instead of `git diff master branch`. I.e. I don't have those diffs between master and my branch, those are diffs between the (poorly chosen) GCA between master and branch and my branch.